internal/sirius/case.go

```diff
@@ -13,6 +14,7 @@ type Case struct {
 	RegistrationDate DateString `json:"registrationDate"`
 	RejectedDate     DateString `json:"rejectedDate"`
 	WithdrawnDate    DateString `json:"withdrawnDate"`
+	Children         []Case     `json:"children"`
```

Comment:
did you mean to add these fields here?

Reply:
Whoops, no, I made this change at the start and then reverted but I think they've come back in a rebase